src/plugins/intel_gpu/src/graph/tile.cpp

```diff
@@ -27,7 +27,7 @@ layout tile_inst::calc_output_layout(tile_node const& node, kernel_impl_params c
     std::vector<int64_t> repeats = desc->repeats;
 
```

Comment:
I think there should be logic here similar to shape infer, to avoid problems with shape mismatch
```suggestion
        const auto output_rank = std::max(arg_shape.size(), repeats->size());

        // add missing repeats
        repeats->insert(repeats->begin(), output_rank - repeats->size(), TDim{1});

        // insert missing input dimensions
        auto rep_it = std::next(repeats->begin(), output_rank - arg_shape.size());
        output_shape.insert(output_shape.begin(), repeats->begin(), rep_it);
```